.eleventy.js

```diff
@@ -357,24 +359,36 @@ module.exports = function (eleventyConfig) {
           ${demoDates
             .map(date => {
               if (validDates.includes(date.date)) {
-                const options = {
+                const dateOptions = {
                   weekday: 'long',
                   year: 'numeric',
                   month: 'long',
                   day: 'numeric',
                   timeZone: 'UTC',
```

Comment:
Another optional comment since this wasn't your code: Instead of manually adding timeZone: 'UTC' to dateOptions, should we make the time zone dynamic based on the user's system preferences?